web/src/layers/NeighbourhoodRoadLayer.svelte

```diff
@@ -80,7 +80,8 @@
   {...layerId("interior-roads-outlines")}
   filter={["==", ["get", "kind"], "interior_road"]}
   paint={{
-    "line-width": lineWidth($thickRoadsForShortcuts, gj.maxShortcuts, 0),
+    // REVIEW: the interior-roads-outline was previously not visible - was that intentional?
```

Comment:
Just opening a comment to remind myself to look at this...

Reply:
Before:
![image](https://github.com/user-attachments/assets/07af9071-c1ec-4cda-8982-df252695bf9d)
After:
![image](https://github.com/user-attachments/assets/cbb1c72e-2017-4f58-8c22-3adcc1257857)

This was a mistake, the outlines should be visible -- I think they help contrast roads and cells. Good catch